cove/input/migrations/0007_supplied_data_schema_version.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('input', '0006_supplieddata_rendered'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='supplieddata',
+            name='schema_version',
+            field=models.CharField(max_length=10),
+        ),
+        migrations.AlterField(
+            model_name='supplieddata',
+            name='rendered',
+            field=models.BooleanField(default=False),
```

Comment:
Need to check if this causes any damage.  This is a migration that should have been run a while ago.